lib-router/src/main/java/com/github/sasd97/lib_router/commands/providers/With.java

```diff
@@ -1,23 +1,24 @@
+// Copyright (c) 2018 by Alexander Dadukin (st235@yandex.ru)
+// All rights reserved.
+
 package com.github.sasd97.lib_router.commands.providers;
 
 import android.os.Bundle;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 
-/**
- * Created by alexander on 06/09/2017.
- */
-
 public final class With extends ProviderCommand<Bundle> {
 
-    protected Bundle bundle;
+    @Nullable
+    private final Bundle bundle;
 
     private With(@NonNull Bundle bundle) {
         this.bundle = bundle;
     }
 
     private With(@NonNull ProviderCommand<Bundle> command) {
         super(command);
+        bundle = null;
```

Comment:
why u need to make bundle null here? U can just delete final from its field

Reply:
without `final` field will be modifiable 
its not true.